scrapers/covid-19/historical.js

```diff
@@ -116,6 +146,30 @@ const getHistoricalCountryDataV2 = (data, query, province = null, lastdays = 30)
 	lastdays = stringUtils.getLastDays(lastdays);
 	const countryInfo = nameUtils.getCountryData(query);
 	const standardizedCountryName = stringUtils.wordsStandardize(countryInfo.country ? countryInfo.country : query);
+	/*
+	  If data required is for a Province/Region of
+	  Canada or data required is for a country other
+	  than Canada, remove the edge case data inserted and
+	  and keep it if general historial data for Canada
+	  is required.
+	*/
+	if (standardizedCountryName.toLowerCase() === 'canada') {
+		if (province !== null) {
+			for (let i = 0; i < data.length; i++) {
+				if (data[i].country.toLowerCase() === 'canada' && data[i].province.toLowerCase() === 'recovered-aggregate') {
+					data.splice(i, 1);
+					break;
+				}
+			}
+		}
+	} else {
+		for (let i = 0; i < data.length; i++) {
+			if (data[i].country.toLowerCase() === 'canada' && data[i].province.toLowerCase() === 'recovered-aggregate') {
+				data.splice(i, 1);
+				break;
+			}
+		}
+	}
```

Comment:
Again, why are we removing here, we won't be using it anyways

Reply:
TBH I don't think the `else` block is necessary if the requested data is not for `Canada`. I don't know why I included it there.